src/styles/index.scss

```
@@ -1,3 +1,235 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+$hover-text-color: #34568b;
+$color: #00acdc;
```

Use a more descriptive name for this color, fix it everywhere 
```suggestion
$main-color: #00acdc;
```